src/streamlit.py

```diff
@@ -166,4 +168,24 @@ def make_game_info_card(games, key=0):
     with col2:
         make_game_info_card(games, 2)
 
-        
\ No newline at end of file
+def ratings_page(ratings_arr):
+    # Main Plot of Regulae
+    plt.style.use('dark_background')
+    fig, ax = plt.subplots()
+    n_games = len(ratings_arr)
+
+    for i in range(n_games):
+        player = ratings_arr[i]
+        if(player.n_games < 5): continue
```

Comment:
we can change this to whatever, but without this it would be polluted by loads of players that hadn't really played